docs/proposals/device-crd.md

```diff
@@ -185,8 +185,55 @@ type VisitorConfig struct {
 	// Modbus represents a set of additional visitor config fields of modbus protocol.
 	// +optional
 	Modbus VisitorConfigModbus `json:"modbus,omitempty"`
+	// Bluetooth represents a set of additional visitor config fields of bluetooth protocol.
+	// +optional
+	Bluetooth VisitorConfigBluetooth `json:"bluetooth,omitempty"`
+}
+
+// Common visitor configurations for bluetooth protocol
+type VisitorConfigBluetooth struct {
+	// Required: Unique ID of the corresponding operation
+	CharacteristicUUID string `json:"characteristic-uuid,omitempty"`
+	//Responsible for converting the data coming from the platform into a form that is understood by the bluetooth device
+	// For example: "ON":[1], "OFF":[0]
+	//+optional
+	BluetoothDataWrite map[string][]byte `json:"write-data,omitempty"`
```

Comment:
Camel naming please.

Reply:
@m1093782566 sry, i didn't get it....."BluetoothDataWrite" is this exported variable not in camel case already?

Reply:
I mean "write-data"

Reply:
oh...got it, thanks. Done